### DISCUSSION TITLE
Inconsistent actions in different views

### DISCUSSION THREAD

**Discussion by lukaszwawrzyk** [immich-app/immich#14930] [Feature Request]
### The bug

Immich has multiple views like timeline, albums, etc. and photos can be selected or multiselected or opened for focused asset view.
I think that all actions that make sense should be consistently available in all possible views. Maybe ordering can be different, maybe less common actions can be in overflow menu or whatever, but I believe these should be available.
To add some example:
In album asset view it is not possible to delete photo, but it is possible with multiselect.

### The OS that Immich Server is running on

.

### Version of Immich Server

v1.123.0

### Version of Immich Mobile App

v1.123.0

### Platform with the issue

- [ ] Server
- [X] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
.
```

### Your .env content

```Shell
.
```

### Reproduction steps

.

**Comment by darrendavid** [comment]
+1 for this. I seem to be unable to create a stack of images in the web app from within an album.

**Comment by GikikCrowchew** [comment]
Hey All,
I posted something in March and it gained several reactions asking for same, and recentlye @clinta closed it probably duplicate, seems simar to this issue !(https://github.com/immich-app/immich/discussions/15022#discussioncomment-12393738)):
The interface of Tags is completely different from for example Albums or timeline.
You should make the menu bar uniform, so that.
![Interface-Albums](https://github.com/user-attachments/assets/731a368b-0c3a-4e2e-9211-ac8d61601ec2)
![Interface-Tags](https://github.com/user-attachments/assets/c59ce28a-e15d-4eb3-a321-a262154d1a3a)
When viewing details of an image, you should be able to remove from album, just like remove a tag.
Would also be great that you can easily assign several images (or all images with certain tag or album) to another tag or album.
> I mean it should be uniform bar from timeline or album view.
> Seen as total solution: The "detail view" of an image is uniform, so a "collection view" should be same button layout acros all collection views (so for tag selection -view, master timeline - view, people collection - view, album collection view and places collection view.
> I mean if then the button bar then is adjustable in user-preferences, then we are just entering insane user-comfor teritory

**Comment by MWP** [comment]
It is not possible to move photos to the locked folder from the folders view like it is in the normal timeline view.
The photos in the folders view need to viewed and moved one at a time.

**Reply by alextran1502** [reply]
#27384

**Comment by smileBeda** [comment]
As discussed at https://discord.com/channels/979116623879368755/994044917355663450/1528760064368705558 - locked folder items completely miss any options other than share.
This means you also can’t edit locked folder items.

Thus reconciling the different views so they all show the same buttons and options as sensible should include edit options on locked folder.
